sdcm/tester.py

```diff
@@ -290,6 +280,21 @@ def setUp(self):
 
         self.db_cluster.validate_seeds_on_all_nodes()
 
+    def set_system_auth_rf(self):
+        # change RF of system_auth
+        system_auth_rf = self.params.get('system_auth_rf', default=3)
```

Comment:
we discussed, no params defaults in the code!!!!!

Reply:
Adding to test_default.yaml